superbench/runner/ansible.py

```diff
@@ -78,6 +78,20 @@ def run(self, ansible_config, sudo=False):    # pragma: no cover
         logger.info(r.stats)
         return r.rc
 
+    def update_mpi_config(self, ansible_config):
+        """Update ansible config for mpi.
+
+        Args:
+            ansible_config (dict): Ansible config dict.
+
+        Returns:
+            dict: Updated Ansible config dict.
+        """
+        if ansible_config['host_pattern'] != 'localhost':
```

Comment:
Why is `localhost` different from others? For example, I give 127.0.0.1. It should be the same, correct?

Reply:
As Yifan mentioned, it's ansible's configuration